common/pb/event_move_card.proto

```diff
@@ -16,4 +16,6 @@ message Event_MoveCard {
     optional sint32 y = 9 [default = -1];
     optional sint32 new_card_id = 10 [default = -1];
     optional bool face_down = 11;
+    optional string uuid = 12;
```

Comment:
looking at the code I only see new_card_uuid being used and not this field

Reply:
@ZeldaZach Do you remember why we added this as well?
It was something about parity between id and new_card_id but I don't remember the exact reasoning.